docs/en/sql-reference/sql-functions/array-functions/array_sortby.md

```diff
@@ -27,6 +27,7 @@ After array `b` is sorted in ascending order, it becomes [5,6,7]. Array `a` beco
 ```Haskell
 array_sortby(array0, array1)
 array_sortby(<lambda function>, array0 [, array1...])
+array_sortby(array0, array1, array2 [, array3...])
```

Comment:
is `array2` necessary?
how about `array_sortby(array0, array1, [array2, array3...])`?